### ISSUE TITLE
Conda packages doesn't leave the extensions in $prefix/share/jupyter/nbextensions

### ISSUE THREAD

**Issue by maartenbreddels** [conda-forge/pythreejs-feedstock#19]
```
$ conda create -c conda-forge -n pythreejs pythreejs
$ ls ~/miniconda3/envs/pythreejs/share/jupyter/nbextensions                                                         pythreejs
jupyter-datawidgets jupyter-js-widgets  jupyter-scales
# Note the missing jupyter-threejs
$ conda uninstall pythreejs
$ pip install pythreejs
$ ls ~/miniconda3/envs/pythreejs/share/jupyter/nbextensions                                                         pythreejs
jupyter-datawidgets jupyter-js-widgets  jupyter-scales      jupyter-threejs
# but pip installs it
```

**Comment by vidartf** [comment]
This has been fixed for 1.1.0, but unfortunately, the OSX build is not currently completing. I'll ping/close this issue when that is fixed.

**Comment by vidartf** [comment]
This should be fixed in the most recent build (1.1.0-2).
